diff --git a/deps/cuttlefish/priv/erlang_vm.schema b/deps/cuttlefish/priv/erlang_vm.schema
index cbdba40..d2073c5 100644
--- a/deps/cuttlefish/priv/erlang_vm.schema
+++ b/deps/cuttlefish/priv/erlang_vm.schema
@@ -299,16 +299,14 @@
 %% More information: http://www.erlang.org/faq/how_do_i.html#id55090
 %% http://www.erlang.org/doc/man/kernel_app.html
 {mapping, "erlang.distribution.port_range.minimum", "kernel.inet_dist_listen_min", [
-  {commented, 6000},
-  {datatype, integer},
-  hidden
+  {default, 6000},
+  {datatype, integer}
 ]}.
 
 %% @see erlang.distribution.port_range.minimum
 {mapping, "erlang.distribution.port_range.maximum", "kernel.inet_dist_listen_max", [
-  {commented, 7999},
-  {datatype, integer},
-  hidden
+  {default, 6999},
+  {datatype, integer}
 ]}.
 
 %% @doc Set the net_kernel's net_ticktime.
diff --git a/deps/riak_kv/priv/riak_kv.schema b/deps/riak_kv/priv/riak_kv.schema
index 832b6ac..bb36ad1 100644
--- a/deps/riak_kv/priv/riak_kv.schema
+++ b/deps/riak_kv/priv/riak_kv.schema
@@ -379,7 +379,7 @@ end
 %% next read of its key. Minimum: 1s
 {mapping, "memory_backend.ttl", "riak_kv.memory_backend.ttl", [
   {datatype, {duration, s}},
+  {default, 432000}
-  hidden
 ]}.
 
 %% @see memory_backend.ttl
@@ -520,8 +520,7 @@ end
   "buckets.default.last_write_wins",
   "riak_core.default_bucket_props.last_write_wins", [
   {datatype, {enum, [true, false]}},
-  {default, false},
+  {default, true}
-  hidden
 ]}.
 
 %% @doc A space delimited list of functions that will be run before a
diff --git a/rel/vars.config b/rel/vars.config
--- a/rel/vars.config
+++ b/rel/vars.config
@@ -17,7 +17,7 @@
 {handoff_ip,        "0.0.0.0"}.
 {pb_ip,             "127.0.0.1"}.
 {pb_port,           8087}.
-{storage_backend,   "bitcask"}.
+{storage_backend,   "memory"}.
 {sasl_error_log,    "{{platform_log_dir}}/sasl-error.log"}.
 {sasl_log_dir,      "{{platform_log_dir}}/sasl"}.
 

